README.md

```diff
@@ -1 +1,146 @@
-# scalardl-web-client-sdk
+## Scalar DL Web Client SDK
+
+This is a client-side library for web applications to interact with [Scalar DL](https://github.com/scalar-labs/scalardl) network.
+
+## Install
+
+We can use package managers to install this library.
+
+**NPM**
+```
+npm install @scalar-labs/scalardl-web-sdk
```

Comment:
Let's name it `scalardl-web-client-sdk`.
Please update the related ones also.
Sorry for saying this here.

Reply:
Understood.